doc/classes/ImmediateGeometry.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ImmediateGeometry" inherits="GeometryInstance" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Node to draw simple geometry from code, ala OpenGL 1.x
+		Node to draw simple geometry from code, ala OpenGL 1.x.
 	</brief_description>
 	<description>
+		Node to draw simple geometry from code, ala OpenGL 1.x.
```

Comment:
There is no benefit in having this kind of description, it just repeats the same as brief.

Reply:
Was decided like that on Discord channel. @cbscribe @NathanLovato ?